Gcpe.Hub.API/Controllers/MessagesController.cs

```diff
@@ -101,24 +91,26 @@ public IActionResult AddMessage(Models.Message message)
             }
         }
 
-        [HttpGet("{id}", Name = "GetMessage")]
-        [Produces(typeof(Models.Message))]
-        [ProducesResponseType(400)]
-        [ProducesResponseType(404)]
-        public IActionResult GetMessage(Guid id)
+        private void AdjustMessage(Message dbMessage, bool wasPublished = false)

```

Comment:
the name of this method is kind of generic and does a two things: bumping other messages down when a newly published message is created and removing old highlights. These seem like two separate things to me so I think they should be in two different methods. Having the two different ways of bumping other messages sort orders around is kinda confusing but I can live with that..